stable/atlantis/README.md

```diff
@@ -24,6 +24,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 
 | Parameter                                   | Description                                                                                                                                                                                                                                                                                               | Default |
 |---------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `allowRepoConfig`                           | Whether to allow the use of [atlantis.yaml files](https://www.runatlantis.io/docs/atlantis-yaml-reference.html).                                                                                                                                                                                          | `false` |
```

Comment:
So you're adding all of these back now which were removed by https://github.com/helm/charts/pull/13253. Please rebase and ensure this changeset only has your changes.